CMakeLists.txt

```diff
@@ -92,6 +92,10 @@ set(xeus_REQUIRED_VERSION 5.0.0)
 set(xeus_zmq_REQUIRED_VERSION 3.0.0)
 set(CppInterOp_REQUIRED_VERSION 1.7.0)
 
+if(EMSCRIPTEN)
+    set(xeus_lite_REQUIRED_VERSION 3.2.1)
+endif()
```

Comment:
I think we can set `xeus_lite_REQUIRED_VERSION` regardless of whether we are in EMSCRIPTEN.